content/blog/2019/02/2019-02-08-windows-installers.adoc

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Windows Installer Updates"
+tags:
+- jenkins

```

Comment:
Maybe not needed. It is always about Jenkins in this blog

Reply:
I didn't find a recommended list of tags, so I left it pretty general. The other tags you suggest are good.